homeassistant/components/kef/media_player.py

```diff
@@ -210,7 +208,16 @@ def is_volume_muted(self):
     @property
     def supported_features(self):
         """Flag media player features that are supported."""
-        return SUPPORT_KEF
+        support_kef = (
+            SUPPORT_VOLUME_SET
+            | SUPPORT_VOLUME_STEP
+            | SUPPORT_VOLUME_MUTE
+            | SUPPORT_SELECT_SOURCE
+        )
+        if not self._supports_on_off:
+            return support_kef
+        else:
+            return support_kef | SUPPORT_TURN_OFF | SUPPORT_TURN_ON
```

Comment:
Modify `support_kef` here instead so we only need to return once.

Reply:
Done